2-Authorization-I/1-call-graph/App/signout.html

```diff
@@ -0,0 +1,19 @@
+<!DOCTYPE html>
```

Comment:
we can leave this out, this is concept is relevant to ch 1 mostly

Reply:
Removed!